WORKSPACE

```diff
@@ -1,9 +1,8 @@
-new_http_archive(
+new_git_repository(
```

Comment:
there is a problem with the packaging of the remote repo. It contains an invalid python file.